src/xpcc/architecture/platform/devices/stm32/stm32f103-c_r_t_v-8_b.xml

```diff
@@ -0,0 +1,388 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE rca SYSTEM "../devicefile.dtd">
+<rca version="1.0">
+  <!-- WARNING: This file is generated automatically, do not edit!
+ 		Please modify the xpcc/tools/device_file_generator code instead and rebuild this file.
+ 		Be aware, that regenerated files might have a different composition due to technical reasons. -->
+  <device platform="stm32" family="f1" name="103" pin_id="c|r|t|v" size_id="8|b">
+    <flash>65536</flash>
```

Comment:
Yeah, found a bug in the stm32 device file reader. It will be fixed in a separate commit though.